internal/relay/pinpoint/relay.go

```diff
@@ -0,0 +1,75 @@
+package relay
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/pinpointemail"
+	"github.com/aws/aws-sdk-go/service/pinpointemail/pinpointemailiface"
+)
+
+// Request object
+type Request struct {
+	Time  time.Time
+	addr  net.Addr
+	IP    string
+	From  string
+	To    []string
+	Error string
+	err   *error
+}
+
+// Log prints the Request data to Stdout/Stderr
+func (req *Request) Log() {
+	req.IP = req.addr.(*net.TCPAddr).IP.String()
+	var out *os.File
+	err := *req.err
+	if err != nil {
+		req.Error = err.Error()
+		out = os.Stderr
+	} else {
+		out = os.Stdout
+	}
+	b, _ := json.Marshal(req)
+	fmt.Fprintln(out, string(b))
+}
```

Comment:
Since this is exactly the same function as in the SES Relay, this should be in a shared library.

Reply:
👍 Done.